src/components/portrait/portrait-initials.component.tsx

```diff
@@ -1,23 +1,37 @@
 import React, { useEffect, useState } from "react";
-import PropTypes from "prop-types";
 import tokens from "@sage/design-tokens/js/base/common";
 
+import { PortraitSizes, PortraitShapes } from "./portrait.component";
 import {
   StyledPortraitInitials,
   StyledPortraitInitialsImg,
   getColorsForInitials,
 } from "./portrait.style";
 import { PORTRAIT_SIZE_PARAMS } from "./portrait.config";
 
+export interface PortraitInitialsProps {
+  /** The user's initials to render. */
+  initials: string;
+  /** The size of the initials image. */
+  size: PortraitSizes;
+  /** Use a dark background. */
+  darkBackground?: boolean;
+  /** The shape of the Portrait. */
+  shape?: PortraitShapes;
+  /** The `alt` HTML string. */
+  alt?: string;
+}
+
 const PortraitInitials = ({
   initials,
   size,
-  shape,
+  shape = "square",
   darkBackground,
   alt,
-  ...rest
-}) => {
-  const [cachedImageDataUrl, setCachedImageDataUrl] = useState();
+}: PortraitInitialsProps) => {
+  const [cachedImageDataUrl, setCachedImageDataUrl] = useState<string | null>(
+    null
+  );
```

Comment:
I think that you could also refactor `setCachedImageDataUrl(null);` to `setCachedImageDataUrl("");`, then you could drop the null type here. IMO as we are refactoring to TypeScript, we should avoid changing types in our variables.

```suggestion
  const [cachedImageDataUrl, setCachedImageDataUrl] = useState<string>();
```